modular_skyrat/modules/death_consequences_perk/death_consequences_customization.dm

```diff
@@ -0,0 +1,228 @@
+
+	minimum = 0
+	maximum = 1000
+
+/datum/preference/numeric/death_consequences/degradation_on_death/create_default_value()
+	return DEATH_CONSEQUENCES_DEFAULT_DEGRADATION_ON_DEATH
+
+/datum/preference/numeric/death_consequences/formeldahyde_dead_degradation_mult
+	savefile_key = "dc_formeldahyde_dead_degradation_mult"
+
+	minimum = 0
+	maximum = 1
+
+/datum/preference/numeric/death_consequences/formeldahyde_dead_degradation_mult/create_default_value()
+	return 0
+
+/datum/preference/numeric/death_consequences/stasis_dead_degradation_mult
+	savefile_key = "dc_stasis_dead_degradation_mult"
+
+	minimum = 0
+	maximum = 1
+
+/datum/preference/numeric/death_consequences/stasis_dead_degradation_mult/create_default_value()
+	return 0
+
+/datum/preference/numeric/death_consequences/rezadone_living_degradation_reduction
+	savefile_key = "dc_rezadone_living_degradation_reduction"
+
+	minimum = 0
+	maximum = 500
+
+/datum/preference/numeric/death_consequences/rezadone_living_degradation_reduction/create_default_value()
+	return DEATH_CONSEQUENCES_DEFAULT_REZADONE_DEGRADATION_REDUCTION
+
+/datum/preference/numeric/death_consequences/strange_reagent_degradation_reduction
+	savefile_key = "dc_strange_reagent_degradation_reduction"
+
+	minimum = 0
+	maximum = 500
+
+/datum/preference/numeric/death_consequences/strange_reagent_degradation_reduction/create_default_value()
+	return DEATH_CONSEQUENCES_DEFAULT_STRANGE_REAGENT_DEGRADATION_REDUCTION
+
+/datum/preference/numeric/death_consequences/sansufentanyl_living_degradation_reduction
+	savefile_key = "dc_sansufentanyl_living_degradation_reduction"
+
+	minimum = 0
+	maximum = 500
+
+/datum/preference/numeric/death_consequences/sansufentanyl_living_degradation_reduction/create_default_value()
+	return DEATH_CONSEQUENCES_DEFAULT_SANSUFENTANYL_DEGRADATION_REDUCTION
+
+/datum/preference/numeric/death_consequences/eigenstasium_degradation_reduction
+	savefile_key = "dc_eigenstasium_degradation_reduction"
+
+	minimum = 0
+	maximum = 5000
+
+/datum/preference/numeric/death_consequences/eigenstasium_degradation_reduction/create_default_value()
+	return DEATH_CONSEQUENCES_DEFAULT_EIGENSTASIUM_DEGRADATION_REDUCTION
+
+/datum/preference/numeric/death_consequences/crit_threshold_reduction_min_percent_of_max
+	savefile_key = "dc_crit_threshold_reduction_min_percent_of_max"
+
+	minimum = 0
+	maximum = 100
+
+/datum/preference/numeric/death_consequences/crit_threshold_reduction_min_percent_of_max/create_default_value()
+	return 0 // percent
+
+/datum/preference/numeric/death_consequences/crit_threshold_reduction_percent_of_max
+	savefile_key = "dc_crit_threshold_reduction_percent_of_max"
+
+	minimum = 0
+	maximum = 100
+
+/datum/preference/numeric/death_consequences/crit_threshold_reduction_percent_of_max/create_default_value()
+	return 100 // percent
+
+/datum/preference/numeric/death_consequences/max_crit_threshold_reduction
+	savefile_key = "dc_max_crit_threshold_reduction"
+
+	minimum = 0
+	maximum = MAX_LIVING_HEALTH
```

Comment:
There's SO many prefs for this, it's almost feeling excessive at this point.

Reply:
thats the price you pay for customizability